orion/sites/admin/src/index.js

```diff
@@ -1,72 +0,0 @@
-'use strict';
```

Comment:
I think we still need this file :)

Reply:
ooh I think this happened b/c I deleted my entire file off github since I accidentally uploaded it